nixup/modules/nixup/switch-to-configuration.pl

```diff
@@ -0,0 +1,458 @@
+#! @perl@
```

Comment:
C++ 
See this: https://www.bountysource.com/issues/4399291-get-rid-of-the-perl-dependency and https://github.com/NixOS/nix/issues/341